File: VB/15_CommentAndNote/ReplyToComment.vb
Imports Spire.Presentation

Namespace ReplyToComment
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create ppt file
			Dim ppt As New Presentation()

			'Create Comment author
			Dim author As ICommentAuthor = ppt.CommentAuthors.AddAuthor("E-iceblue", "comment")

			'Add comment
			ppt.Slides(0).AddComment(author, "Add comment", New Point(18, 25), Date.Now)
			Dim comment As Comment = ppt.Slides(0).Comments(0)

			'Add reply to Comment
			If Not comment.IsReply Then
				comment.Reply(author, "Add Reply1", Date.Now)
				comment.Reply(author, "Add Reply2", Date.Now)
			End If

			'delete first reply
			ppt.Slides(0).DeleteComment(author, "Add Reply1")

			'Save the result ppt file
			ppt.SaveToFile("AddReplyToComment.pptx", FileFormat.Pptx2013)
			Process.Start("AddReplyToComment.pptx")
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load

		End Sub
	End Class
End Namespace

File: CS/15_CommentAndNote/ReplyToComment.cs
using Spire.Presentation;
using System;
using System.Drawing;
using System.Windows.Forms;

namespace ReplyToComment
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create ppt file
            Presentation ppt = new Presentation();        

            //Create Comment author
            ICommentAuthor author = ppt.CommentAuthors.AddAuthor("E-iceblue", "comment");

            //Add comment
            ppt.Slides[0].AddComment(author, "Add comment", new System.Drawing.Point(18, 25), DateTime.Now);
            Comment comment = ppt.Slides[0].Comments[0];

            //Add reply to Comment
            if (!comment.IsReply)
            {
                comment.Reply(author, "Add Reply1", DateTime.Now);
                comment.Reply(author, "Add Reply2", DateTime.Now);
            }

            //delete first reply
            ppt.Slides[0].DeleteComment(author, "Add Reply1");

            //Save the result ppt file
            ppt.SaveToFile(@"AddReplyToComment.pptx", FileFormat.Pptx2013);
            System.Diagnostics.Process.Start("AddReplyToComment.pptx");
        }

        private void Form1_Load(object sender, EventArgs e)
        {

        }
    }
}